src/utilities/getSquaredNumber.js

```diff
@@ -0,0 +1,3 @@
+export function getSquaredNumber(arrayOfNumbers) {
```

Comment:
This is the wrong name for this function. Please take a closer look at the exercise.